schemas/iso19115-3.2018/src/main/plugin/iso19115-3.2018/formatter/xsl-view/view.xsl

```diff
@@ -253,7 +253,8 @@
           <img data-gn-img-modal="md"
                class="gn-img-thumbnail center-block"
                alt="{$schemaStrings/overview}"
-               src="{mcc:fileName/*}"/>
+               src="{mcc:fileName/*}"
+               onerror="this.onerror=null; $('.gn-md-side-overview').hide()"/>
```

Comment:
We may have one overview visible and another one not. Can't we do the same as in angular app `this.style.display='none';` to only hide the current image instead of all the panel ? But we may end up with an empty panel.

Reply:
or we check if there is only one overview, and we hide the panel, if more than one we hide only the current image ?

Reply:
I wanted to hide the header, but you are right, could have multiple thumbnails and some could work